src/AutoBuild.jl

```diff
@@ -1258,6 +1270,17 @@ function build_jll_package(src_name::String,
         if lazy_artifacts
             println(io, "using LazyArtifacts")
         end
+
+        if !isempty(augment_platform_block)
+            print(io, """
+                $(augment_platform_block)
+            """)
```

Comment:
I'd rather we write this out to `.pkg/augment_platform.jl`, and then conditionally `include()` it.  And can you also write out a `select_artifacts.jl` file that is [like this one] (https://github.com/JuliaLang/Pkg.jl/blob/2dd335da3dddf492ea03d765061ca7fd6e6c65c2/test/test_packages/AugmentedPlatform/.pkg/select_artifacts.jl).  That will future-proof us for `Pkg.add()`-time detection.

Reply:
What would be the include condition?

Reply:
`if isfile()`?